expyfun/stimuli/_tracker.py

```diff
@@ -820,9 +820,9 @@ def respond(self, correct):
         if self.stopped:
             self._callback(
                 'dealer_%i_stop' % self._dealer_id, json.dumps(dict(
-                    tracker_position=[int(s) for s in self._tracker_history],
-                    responses=[int(s) for s in self._response_history],
-                    x=[int(s) for s in self._x_history])))
+                    tracker_history=[int(s) for s in self._tracker_history],
+                    response_history=[int(s) for s in self._response_history],
+                    x_history=[float(s) for s in self._x_history])))
```

Comment:
Hmmmmmm. Perhaps we should set ``x_current`` to ``np.nan`` rather than ``None`` for ``Tracker_Binom``. It doesn't matter too much, since the point is that ``Tracker_Binom`` doesn't actually use that paramter.